seedDB.js

```diff
@@ -0,0 +1,39 @@
+const db = require('./database/database.js');
+const bn = require('./bandNames.js');
```

Comment:
vowels are free, consonants too. Please use coherent variable names by anyone reading the code below, especially on subsequent pull requests where this specific line won't be in the diff.